### PR TITLE
{lib}[GCCcore/11.3.0] json-fortran v8.3.0

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/j/json-fortran/json-fortran-8.3.0-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+checksums = ['5fe9ad709a726416cec986886503e0526419742e288c4e43f63c1c22026d1e8a']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.24.3'),
+]
+
+configopts = [
+    '-DUSE_GNU_INSTALL_CONVENTION=TRUE',
+]
```
Comment (jfgrimm) [inline easybuild/easyconfigs/j/json-fortran/json-fortran-8.3.0-GCCcore-11.3.0.eb:24]: no need for this to be a list?
```suggestion
configopts = '-DUSE_GNU_INSTALL_CONVENTION=TRUE'
```

**Comment by sassy-crick** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/0339038b8c6dfbf05a31e9725531de28 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**